code/modules/projectiles/guns/magnetic/bore.dm

```diff
@@ -98,7 +98,16 @@
 			capacitor = null
 			update_icon()
 			return
-
+			if(!manipulator)
+				to_chat(user, "<span class='warning'>\The [src] has no manipulator installed.</span>")
+				return
+			manipulator.forceMove(get_turf(src))
+			user.put_in_hands(manipulator)
+			user.visible_message("<span class='notice'>\The [user] unscrews \the [manipulator] from \the [src].</span>")
+			playsound(loc, 'sound/items/Screwdriver.ogg', 50, 1)
+			manipulator = null
+			update_icon()
+			return
```

Comment:
This is going to be completely skipped due to the above return on `line 100`, you'd probably want to move this into its own `if`, perhaps with a crowbar check as on `line 90`, so that each tool removes a different component specifically.